Src/FluentAssertions/Formatting/DefaultValueFormatter.cs

```diff
@@ -9,12 +9,9 @@ namespace FluentAssertions.Formatting
 {
     public class DefaultValueFormatter : IValueFormatter
     {
-        #region Private Definitions
-
         private const int RootLevel = 0;
-        private const int SpacesPerIndentionLevel = 3;
 
-        #endregion
+        protected virtual int SpacesPerIndentionLevel { get; } = 3;
```

Comment:
Can we document all public and protected properties?

Reply:
Of course, I'll add that.